docs/assets/dcr_tmux_simnet_setup.sh

```diff
@@ -0,0 +1,275 @@
+WALLET_MINING_ADDR="SspUvSyDGSzvPz2NfdZ5LW15uq6rmuGZyhL" # NOTE: This must be changed if the seed is changed.
+WALLET_XFER_ADDR="SsonWQK6xkLSYm7VttCddHWkWWhETFMdR4Y" # same as above
+
+if [ -d "${NODES_ROOT}" ] ; then
+  rm -R "${NODES_ROOT}"
+fi
+
+PRIMARY_DCRD_NAME=dcrd1
+SECONDARY_DCRD_NAME=dcrd2
+PRIMARY_WALLET_NAME=wallet1
+SECONDARY_WALLET_NAME=wallet2
+mkdir -p "${NODES_ROOT}/${PRIMARY_DCRD_NAME}"
+mkdir -p "${NODES_ROOT}/${SECONDARY_DCRD_NAME}"
+mkdir -p "${NODES_ROOT}/${PRIMARY_WALLET_NAME}"
+mkdir -p "${NODES_ROOT}/${SECONDARY_WALLET_NAME}"
+
+cat > "${NODES_ROOT}/dcrd.conf" <<EOF
+rpcuser=${RPCUSER}
+rpcpass=${RPCPASS}
+simnet=1
+logdir=./log
+datadir=./data
+debuglevel=TXMP=debug,MINR=debug
+txindex=1
+EOF
+
+cat > "${NODES_ROOT}/dcrctl.conf" <<EOF
+rpcuser=${RPCUSER}
+rpcpass=${RPCPASS}
+simnet=1
+EOF
+
+cat > "${NODES_ROOT}/wallet.conf" <<EOF
+username=${RPCUSER}
+password=${RPCPASS}
+simnet=1
+logdir=./log
+appdata=./data
+pass=123
+enablevoting=1
+EOF
+
+cd ${NODES_ROOT} && tmux -2 new-session -d -s $SESSION
+
+################################################################################
+# Setup the primary dcrd node
+################################################################################
+
+PRIMARY_DCRD_P2P=127.0.0.1:19555
+PRIMARY_DCRD_RPC=127.0.0.1:19556
+tmux rename-window -t $SESSION:0 "${PRIMARY_DCRD_NAME}"
+tmux split-window -v
+tmux select-pane -t 0
+tmux send-keys "cd ${NODES_ROOT}/${PRIMARY_DCRD_NAME}" C-m
+tmux send-keys "dcrd -C ../dcrd.conf --listen ${PRIMARY_DCRD_P2P} --miningaddr=${WALLET_MINING_ADDR}" C-m
+tmux resize-pane -D 15
+tmux select-pane -t 1
+tmux send-keys "cd ${NODES_ROOT}/${PRIMARY_DCRD_NAME}" C-m
+
+cat > "${NODES_ROOT}/${PRIMARY_DCRD_NAME}/ctl" <<EOF
+#!/usr/bin/env bash
+dcrctl -C ../dcrctl.conf \$*
+EOF
+chmod +x "${NODES_ROOT}/${PRIMARY_DCRD_NAME}/ctl"
+
+# Script to mine a specified number of blocks with a delay in between them
+# Defaults to 1 block
+cat > "${NODES_ROOT}/${PRIMARY_DCRD_NAME}/mine" <<EOF
+#!/usr/bin/env bash
+NUM=1
+case \$1 in
+    ''|*[!0-9]*)  ;;
+    *) NUM=\$1 ;;
+esac
+
+for i in \$(seq \$NUM) ; do
+  ./ctl generate 1
+  sleep 1
+done
+EOF
+chmod +x "${NODES_ROOT}/${PRIMARY_DCRD_NAME}/mine"
+sleep 3
+tmux send-keys "./ctl generate 32" C-m
+
+################################################################################
+# Setup the primary wallet
+################################################################################
+
+PRIMARY_WALLET_RPC=127.0.0.1:19557
+PRIMARY_WALLET_GRPC=127.0.0.1:19558
+tmux new-window -t $SESSION:1 -n "${PRIMARY_WALLET_NAME}"
+tmux split-window -v
+tmux select-pane -t 0
+tmux resize-pane -D 15
+tmux send-keys "cd ${NODES_ROOT}/${PRIMARY_WALLET_NAME}" C-m
+tmux send-keys "dcrwallet -C ../wallet.conf --create" C-m
+sleep 2
+tmux send-keys "123" C-m "123" C-m "n" C-m "y" C-m
+sleep 1
+tmux send-keys "${WALLET_SEED}" C-m C-m
+tmux send-keys "dcrwallet -C ../wallet.conf --enableticketbuyer" C-m
```

Comment:
I suggest using `ticketbuyer.limit = 5` so that the wallet doesn't buy an outrageous number of tickets and blocks and logs are smaller and easier to reason about.

Reply:
I tried this at one point, but the issue was that any time a block is missed, because there are always 5 votes in every block, it _always_ leads to depletion of the ticket pool.  Perhaps limiting it to 10 would work well though.  I can give that some testing.

Reply:
Same here.  I changed the dcrdex harness to 6 and it can run indefinitely it seem.

Reply:
In theory, 6 should work, but I believe it's due to the nature of the typical testing here where it is typical to mine 100s of blocks in a row, the wallet still sometimes has trouble keeping up, even with a 1s delay in between, and it ultimately leads to depletion.  It seems that 10 is working reasonably well, so I'll go with that for now.

Long term, it would better to add some logic into the template generator that recognizes when it's on simnet and give it the same treatment as votes where it delays producing the template until either X time passes or there are at least 5 ticket purchases in the block, whichever happens sooner.

Reply:
Updated to set a limit of 10.